BUILD

```diff
@@ -1 +1,15 @@
-package_json(name="hello_pkg")
+package_json(
+    name="hello_pkg",
+    scripts=[
+        node_test_script(
```

Comment:
Side note - `node_test_script` and `node_build_script` don't show up in the documentation under the `package_json` target - is there a way we can make that happen?

Reply:
Good question! We have a similar situation with https://www.pantsbuild.org/2.19/reference/targets/python_distribution#provides : it is a target field whose value is created by a call to `python_artifact()`. In that case it looks like we manually add a link in that field's docs to [relevant guide docs](https://www.pantsbuild.org/2.19/docs/writing-plugins/common-plugin-tasks/custom-python-artifact-kwargs).

So I guess we should do that here too.